src/main/java/Task.java

```diff
@@ -0,0 +1,50 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+    public static int COUNT = 0;
```

Comment:
I think that since the intention here is not to have a constant variable, count should just be in lowercase.